style.css

```diff
@@ -352,57 +384,367 @@ label {
     grid-template-columns: 2fr 2fr;
     justify-content: center;
     margin: 0 auto;
-    ;
   }
 
-  h1 {
-      font-size: 8vw;
-  }
   #headline {
     font-size: 1.5vw;
     line-height: 2;
     height: fit-content;
   }
 
-.hello-text {
-  display: flex;
-  flex-direction: column;
-}
+  .hello-text {
+    display: flex;
+    flex-direction: column;
+    padding-left: 8em;
+    background-image: url(images/project/image_dots.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    height: 93.5vh;
+    justify-content: center;
+  }
 
   .bigimage {
     background-image: url("./images/image-big.png");
-    background-size: 100%;
+    background-size: contain;
+    background-repeat: no-repeat;
   }
 
   .blackBoton {
     width: 254px;
   }
 
+  .see {
+    transition-duration: 0.4s;
+    cursor: pointer;
+    background: #fff;
+    color: #1a2236;
+  }
+
+  .see:hover {
+    background: #1a2236;
+    border-color: #fff;
+    border-width: 1px;
+    color: #fff;
+  }
+
+  .see:active {
+    border-width: 2px;
+  }
+
+  /* project section descktop */
+
+  .projects {
+    margin-top: 15em;
+  }
+
+  .pData {
+    display: none;
+  }
```

Comment:
- [ ] Dear @Feliverse, I see your confusion and I get it. You have done really great so far, unfortunately, I have a concert suggestion following up on the previous changes. Now you have made your display default to `display: none;`, and since you ignore the hover effect your project details are not shown anymore. (`with` or `without` hover) `note: it's hidden on a screen size of >1263px )`. In this case, anyone looking at your project will have a hard time figuring out what the project is about. 
![your-project](https://user-images.githubusercontent.com/69077061/201506613-3a72c4a2-6439-42e2-a3c4-fd308cc504a9.PNG)